update DIME servers

Dhop14 opened the pull request.

cipig commented:
thanks for the PR
50002 seems to be SSL port, not TCP... so the entry also needs `"protocol": "SSL",` param
btw, if you already have a valid SSL cert, then please add WSS to those electrums too, then DIME will also be available in the Web app, not just Mobile and Desktop
eg like this
```
  {
    "url": "electrumx1.dimecoinnetwork.com:50002",
    "ws_url": "electrumx1.dimecoinnetwork.com:50004",
    "protocol": "SSL",
    "contact": [
      { "discord": "whatever" }
    ]
  },
```

Dhop14 commented:
> thanks for the PR 50002 seems to be SSL port, not TCP... so the entry also needs `"protocol": "SSL",` param btw, if you already have a valid SSL cert, then please add WSS to those electrums too, then DIME will also be available in the Web app, not just Mobile and Desktop eg like this
> 
> ```
>   {
>     "url": "electrumx1.dimecoinnetwork.com:50002",
>     "ws_url": "electrumx1.dimecoinnetwork.com:50004",
>     "protocol": "SSL",
>     "contact": [
>       { "discord": "whatever" }
>     ]
>   },
> ```

Thanks for the feedback. Updated the PR to fix the syntax and added WSS for server with SSL

cipig commented:
thanks, but you also need to tell electrumx server to listen on that port, eg:
`SERVICES = tcp://:50001,rpc://:8050,ssl://:50002,wss://:50004`

Dhop14 commented:
> thanks, but you also need to tell electrumx server to listen on that port, eg: `SERVICES = tcp://:50001,rpc://:8050,ssl://:50002,wss://:50004`

Electrumx has been restarted to allow wss, etc. Should be good to go

cipig reviewed (approved):
works fine now, thanks
